tests/logging_helper_test.py

```diff
@@ -457,17 +464,20 @@ def test_zipkin_handler_raises_exception_if_ann_and_bann_not_provided(
             " for foo span" == str(excinfo.value))
 
 
-@mock.patch('py_zipkin.logging_helper.thrift.encode_bytes_list', autospec=True)
 def test_batch_sender_add_span(
-    mock_encode_bytes_list,
     empty_annotations_dict,
     empty_binary_annotations_dict,
     fake_endpoint,
 ):
     # This test verifies it's possible to add 1 span without throwing errors.
     # It also checks that exiting the ZipkinBatchSender context manager
     # triggers a flush of all the already added spans.
-    sender = logging_helper.ZipkinBatchSender(MockTransportHandler())
+    encoder = MockEncoder(encoded_queue='foobar')
```

Comment:
should `encoded_queue` be an iterable of some sort? (ok ok, i know a string is technically an iterable)

Reply:
That's the value that gets returned as if it were an encoded queue. So it should be either a string or a byte array.